### ISSUE TITLE
Update default value of opset_version in torch.onnx.export

### ISSUE THREAD

**Issue by winnietsang** [pytorch/pytorch#48341]
## 🚀 Feature
For API torch.onnx.export, there is an opset_version parameter. 
Is it possible to update the default value from '9' to '13' for this parameter?

## Motivation
Most users may assume if they don't set the opset_version, then their PyTorch model will be convert to the latest stable ONNX version(currently is opset 13). They would only set the opset_version unless they want to export to an older opset version in ONNX. But this is totally not the case, the user will end with a model in opset 9. If they want a newer opset version they need to set that in the API call.

cc @houseroad @spandantiwari @lara-hdr @BowenBao @neginraoof

**Comment by spandantiwari** [comment]
@winnietsang - that's something we have considered, but it would require a breaking change to the API. While I can see merits to this, I am not entirely sure of this. Even if we choose to do this, there will be a deprecation period for this.
@houseroad @jspisak @prasanthpul @faxu @BowenBao @neginraoof for their opinion.

**Comment by jspisak** [comment]
Seems like this could be addressed by a simple prompt to users that enumerates what the latest opset is and, if they are not using it, to update this argument in their code. All of that said, if @spandantiwari and the team here feel like it is worth the BC, i support them..

**Comment by shubhambhokare1** [comment]
@garymm Is this issue related to https://msdata.visualstudio.com/Vienna/_boards/board/t/PyTorch%20ONNX%20exporter/Backlog%20items/?workitem=1468772

**Comment by garymm** [comment]
No, more related to https://msdata.visualstudio.com/Vienna/_workitems/edit/1465266. This issue is blocked by that one, I think. So I'd import this into ADO as a new issue and mark the new issue as successor to 1465266.

**Comment by thiagocrepaldi** [comment]
[Default opset version is 14 now](https://github.com/pytorch/pytorch/blob/master/torch/onnx/_constants.py#L9)
